setup.env

```diff
@@ -1,38 +1,70 @@
-### Change the LSST_SDK_INSTALL and OSPL_HOME to the actual locations and uncomment
+### Script to setup SAL environment. Default config assumes ts_opensplice is
```

Comment:
I am very uneasy calling this a script because it must be sourced (run using the `source` command). I think that makes it a "configuration file".

I think "script" usually means a text file that is executable (because it starts with a suitable #! line and has chmod +x).

Reply:
Changed to configuration file. Yes, it's a sourced script, but lets call it configuration file.